baremetal-prep.md

```diff
@@ -22,6 +22,7 @@ To use baremetal-prep.sh the system needs the following:
 
 - RHEL8 entitled host
 - Two network interfaces: one for provisioning and one for baremetal
+- ansible-repo must be enabled. E.g sudo subscription-manager repos --enable ansible-2.8-for-rhel-8-x86_64-rpms 
```

Comment:
Ansible is currently used to create a host file... I'd say it would be nice to get rid of that dependency by replacing the playbook by something else (a basic `envsubst | ...` or a script that can generate the same file)

Reply:
yeah there was mention somewhere about getting all the assets into one location because the script currently has some cli switches and then the host file that needs to be updated   - the question becomes is it better to have hundreds of switches or just a config file.  Keep in mind the baremetal-prep needs to be aware of the physical host information to prepare the install-config.yaml - back in the preflight.sh days this was all discovered via Redfish but since we are less prescriptive with hardware we need to have the end user provide those details.  How we do it remains a question for meritocracy to answer.

Reply:
I'd put the ansible in as a separate issue, not part if this review.
Unclear if we may use more ansible in the near future, but we we don't make sense to reduce the dependencies.

Reply:
https://github.com/openshift-kni/baremetal-deploy/issues/84